tests/stub/driver_parameters/test_max_connection_pool_size.py

```diff
@@ -9,11 +9,11 @@
 
 class TestMaxConnectionPoolSize(TestkitTestCase):
 
-    required_features = types.Feature.BOLT_4_4,
+    required_features = types.Feature.BOLT_5_0,
 
     def setUp(self):
         super().setUp()
-        self._server = StubServer(9999)
+        self._server = StubServer(9000)
```

Comment:
This port was change to `9999` because of firefox blocking the creation of new connections to `9000` after this test suite, this security block was making javascript driver fails sometimes in random tests (the one which runs after the problematic test). 

The change of port to a port not used by the other tests works around the issue and make the javascript driver PR stable.

Reply:
Ah yes, know I vaguely remember! That's worth a comment! Will do :writing_hand: